Data/src/main/java/org/tribuo/data/columnar/processors/response/BinaryResponseProcessor.java

```diff
@@ -72,7 +164,7 @@ public BinaryResponseProcessor(String fieldName, String positiveResponse, Output
 
     @Override
     public String getFieldName() {
```

Comment:
Should we tag this method deprecated too? I think that means that Maven doesn't warn you for using a deprecated method.